### PR TITLE
Add tito_release and tito_build modules

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
@ekohl I took your PR and rebased (a few conflicts to work through) it to work on getting this in. I have a need that your modules perfectly help solve.

**Comment by ehelms** [comment]
@ekohl Please review the top commit I added. I was running into an issue with srpm build not accepting `--arg` based arguments.

File: obal/data/modules/tito_build.py [hunk 1]
```diff
@@ -36,6 +38,13 @@ def main():
         if value:
             command += ['--' + param, value]
 
+    if module.params['builder']:
```
Comment (ekohl) [inline obal/data/modules/tito_build.py]: You could add `builder` to the tuple on line 36 since this behaves just as `--dist`, `--scl` and `--output`.

**Comment by ehelms** [comment]
Updated to add builder to the tuple.

**Comment by ehelms** [comment]
Happy to do a release... once I remember how

**Comment by ehelms** [comment]
@evgeni Can you help me understand this pylint error? It just started showing up and seems unrelated to my code

**Comment by evgeni** [comment]
Looks like a pylint or pytest bug? Not sure and would have to look in more detail tomorrow. Def not your code :)

**Comment by evgeni** [comment]
@ehelms fun. https://github.com/theforeman/obal/pull/204 fixes the issue, but I still think it's a pylint bug which I will look further into.

**Comment by evgeni** [comment]
pytest-xdist bug, nit pylint. #204 + #206 will fix all of them. enjoy

**Pull request merged**